includes/omise-wc-setting.php

```diff
@@ -70,36 +71,36 @@
 		'label'       => Omise_Card_Image::get_amex_image(),
 		'css'         => Omise_Card_Image::get_css(),
 		'default'     => Omise_Card_Image::get_amex_default_display(),
-		'description' => __( 'This only controls the icons displayed on the checkout page.<br />It is not related to card processing on Omise payment gateway.', $text_domain )
+		'description' => Omise_Util::translate( 'This only controls the icons displayed on the checkout page.<br />It is not related to card processing on Omise payment gateway.' )
 	),
 	'title' => array(
-		'title'       => __( 'Title', $text_domain ),
+		'title'       => Omise_Util::translate( 'Title', 'Setting title' ),
 		'type'        => 'text',
-		'description' => __( 'This controls the title which the user sees during checkout.', $text_domain ),
-		'default'     => __( 'Omise Payment Gateway', $text_domain )
+		'description' => Omise_Util::translate( 'This controls the title which the user sees during checkout.' ),
+		'default'     => Omise_Util::translate( 'Omise Payment Gateway', 'Default title' )
```

Comment:
@nimid from https://github.com/omise/omise-woocommerce/pull/24/files#r74709908
this one should be `Default payment title` instead.